pkg/cmd/server/start/controllers.go

```diff
@@ -53,15 +54,15 @@ func newControllerContext(
 				Namespace:            bootstrappolicy.DefaultOpenShiftInfraNamespace,
 			},
 		},
-		InternalKubeInformers:   informers.internalKubeInformers,
+		InternalKubeInformers:   informers.GetInternalKubeInformers(),
```

Comment:
getters with Get prefix in Go, and so many 😱

Reply:
> getters with Get prefix in Go, and so many

Name conflicts with fields.  Anyway, the guy saying it was a bad prefix is wrong.